### PR TITLE
PACA-963: rda api version 0.13 updates

### PR DESCRIPTION
**Pull request by brianburton**
**JIRA Ticket:**
[PACA-963](https://jira.cms.gov/browse/PACA-963)

**User Story or Bug Summary:**

Modify pipeline RDA API client to use version 0.13.0 that contains new FISS primary key field.

**DO NOT MERGE**: RDA API team needs to deploy 0.13.0 release before this is deployed.

---

### What Does This PR Do?

Updates include:

- New proto files for the 0.13 release.
- Add `intermediaryNb` field and column.
- Update various tests to handle new field.

Notes:

- The new field is not optional in the proto file but the column is still nullable until we can be sure it will be present in all claims when we truncate and reingest prior to launch.
- We are not using the new `rda_claim_key` in `record_source` yet.  We will switch to using it for `claim_id` in a followup PR when we make that change plus truncate the database,

### What Should Reviewers Watch For?

If you're reviewing this PR, please check for these things in particular:

* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [x] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [x] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [x] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [x] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* RDA API team must have deployed their 0.13.0 release to production before this is deployed.

### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] The data dictionary has been updated with any field mapping changes, if any were made.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

**Review by daryll — APPROVED** [review]
lets do this ;-)

File: apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java [hunk 1]
```diff
@@ -494,6 +496,7 @@ public static FissClaim.Builder createDefaultClaimBuilder() {
           .setCurrStatusEnum(FissClaimStatus.CLAIM_STATUS_ROUTING)
           .setCurrTranDtCymd(HARDCODED_TRAN_DATE_CYMD)
           .setFedTaxNb(HARDCODED_FED_TAX_NUMBER)
+          .setIntermediaryNb(HARDCODED_INTERMEDIARY_NB)
```
Comment (KyRobbins) [inline apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java:499]: Do we need to set the claim key too so they can be properly ingested later?

File: apps/bfd-model/bfd-model-rda/mappings/FissClaim.yaml [hunk 2]
```diff
@@ -485,6 +493,8 @@ mappings:
     # TODO: Change from to claim id field once it is available in RDA API message.
```
Comment (KyRobbins) [inline apps/bfd-model/bfd-model-rda/mappings/FissClaim.yaml:493]: Isn't this available now with the 0.13 update?

**Review by KyRobbins — DISMISSED** [review]

File: apps/bfd-model/bfd-model-rda/mappings/FissClaim.yaml (hunk 2 above)
Comment (brianburton) [reply apps/bfd-model/bfd-model-rda/mappings/FissClaim.yaml:493]: Yes and we will start using it later in PACA-752, which is coming up soon.  When we start using it we also need to truncate the db, etc..

File: apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java (hunk 1 above)
Comment (brianburton) [reply apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java:499]: Good point.  I'll add that to the TODO list for PACA-752 when we switch to using it.

**Review by KyRobbins — APPROVED** [review]

**Review by daryll — APPROVED** [review]

**Review by mackec1 — APPROVED** [review]

**Review by mjburling — CHANGES_REQUESTED** [review]
When can we merge/deploy this one, @brianburton?

**Comment by brianburton** [comment]
> When can we merge/deploy this one, @brianburton?

This PR can't be merged until after RDA API team upgrades their production server to 0.13.0.  We need to coordinate that with the creation of a snapshot db for our data analyst.  So... Monday or Tuesday probably.

**Pull request merged**